cmd/kube-proxy/app/options/options.go

```diff
@@ -63,25 +63,25 @@ func NewProxyConfig() *ProxyServerConfig {
 func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.BindAddress}, "bind-address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
-	fs.IntVar(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
+	fs.Int32Var(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
```

Comment:
Yeah, had planned on it.  Wanted to get a total number first.

On Wed, Apr 27, 2016 at 2:46 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In cmd/kube-proxy/app/options/options.go
> <https://github.com/kubernetes/kubernetes/pull/24845#discussion_r61211204>
> :
>
> > @@ -63,25 +63,25 @@ func NewProxyConfig() *ProxyServerConfig {
> >  func (s *ProxyServerConfig) AddFlags(fs *pflag.FlagSet) {
> >  	fs.Var(componentconfig.IPVar{Val: &s.BindAddress}, "bind-address", "The IP address for the proxy server to serve on (set to 0.0.0.0 for all interfaces)")
> >  	fs.StringVar(&s.Master, "master", s.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig)")
> > -	fs.IntVar(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
> > +	fs.Int32Var(&s.HealthzPort, "healthz-port", s.HealthzPort, "The port to bind the health check server. Use 0 to disable.")
>
> @smarterclayton <https://github.com/smarterclayton> - all the changes
> int<->int32 are pretty independent and we can do them very safely.
> Can you please move them to a separate PR - so that we can merge it
> earlier, since this doesn't seem to be controversial.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24845/files/6f558a4578868b1c76783f147b6b34817efa5597#r61211204>
>